specification/semantic_conventions.md

```diff
@@ -26,6 +26,8 @@ All Splunk distributions of OpenTelemetry,
 - SHOULD set the following resource attributes when applicable:
   - `telemetry.auto.version`
 
+- SHOULD set [process an process runtime resource attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.18.0/specification/resource/semantic_conventions/process.md)
```

Comment:
note: The OTel spec is "experimental". However, our distros are already setting these attributes. That is why I put a `SHOULD`. Still I am unsure if this should land in "stable" section or we should it be under "experimental".

Reply:
Now I see that even https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk is marked as `Experimental`...

Reply:
`process an process` (is that right?)

Reply:
typo fixed 😆 it should be `and`.